src/lib/dateHandler.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * The TranslateMonth type
+ */
+type TranslateMonth = (month: number) => string;
+
+/**
+ * DateHandler class
+ */
+export class DateHandler {
```

Comment:
I would stick to exporting methods directly. 
Unless there is some kind of state that you need to store classes make little sense IMO. 

If you would have something like `new DateHandler("dd.mm.yyyy")` and than only call the methods without the format this class approach would make sense. But otherwise it's better to export pure functions instead of a class.